public/assets/js/publishFileFunctions.js

```diff
@@ -1,99 +1,231 @@
-/* publish functions */
+var stagedFiles = null;
 
-function cancelPublish () {
-    window.location.href = '/';
-}
+var publishFileFunctions = {
+    triggerFileChooser: function (fileInputId) {
+        document.getElementById(fileInputId).click();
+    },
+    cancelPublish: function () {
```

Comment:
Late on catching this one, but this function name doesn't imply it is redirecting me to the homepage.